discord/ext/bridge/core.py

```diff
@@ -226,9 +383,14 @@ async def convert(self, ctx, argument):
         except BadArgument:
             return await UserConverter().convert(ctx, argument)
 
-
-def attachment_callback(*args):  # pylint: disable=unused-argument
-    raise ValueError("Attachments are not supported for bridge commands.")
+class AttachmentConverter(Converter):
+    async def convert(self, ctx: Context, arg: str):
+        try:
+            attach = ctx.message.attachments[0]
+        except KeyError:
```

Comment:
The exception is incorrect
```suggestion
        except IndexError:
```